resources/europe/greece/gr-matrix.json

```
@@ -4,7 +4,7 @@
   "locationSet": {"include": ["gr"]},
   "languageCodes": ["el"],
   "name": "OpenStreetMap Greece Matrix",
-  "description": "Join #osm-gr:matrix.org at https://riot.im/app/#/room/%23osm-gr:matrix.org",
-  "url": "https://riot.im/app/#/room/%23osm-gr:matrix.org",
+  "description": "Join #osm-gr:matrix.org at https://matrix.to/#/#osm-gr:matrix.org",
```

This can use a replacement token to make translations easier.
```suggestion
  "description": "Join #osm-gr:matrix.org at {url}",
```